src/server/rdb_load.cc

```diff
@@ -1260,7 +1381,19 @@ error_code RdbLoaderBase::ReadObj(int rdbtype, OpaqueObj* dest) {
       iores = ReadStreams();
       break;
     case RDB_TYPE_JSON:
-      iores = ReadJson();
+    case RDB_TYPE_SET_LISTPACK:
+      // We need to deal with protocol versions 9 and older because in these
```

Comment:
please mention that *Dragonfly* defined `RDB_TYPE_JSON` as 20 and it later overlapped with Redis definitions

Reply:
Yes it's not apparent from the comment, I agree. Will update this :)